src/main/java/makePost/use_case/MakePostInputBoundary.java

```diff
@@ -0,0 +1,22 @@
+package makePost.use_case;
+
+import entities.Post;
+import makePost.interface_adapters.MakePostViewModel;
+
+import java.util.List;
+import java.util.Map;
+
+public interface MakePostInputBoundary {
+    void makePost(MakePostRequestModel mprm);
+    int getNumPostsCreated();
+    int getCurrentUser();
+    List<Map<String, Object>> getFavouritedUsers();
```

Comment:
Not sure if this also applies here but I thought I'd ask about the same concern as the comment in `MakePostDataAccessInterface`.